Hardcore.lua

```diff
@@ -22,13 +22,28 @@ local GRIEF_WARNING_OFF = 0
 local GRIEF_WARNING_SAME_FACTION = 1
 local GRIEF_WARNING_ENEMY_FACTION = 2
 local GRIEF_WARNING_BOTH_FACTIONS = 3
+local CLASSES = {
+	-- Classic:
+	[1] = "Warrior",
+	[2] = "Paladin",
+	[3] = "Hunter",
+	[4] = "Rogue",
+	[5] = "Priest",
+	[7] = "Shaman",
+	[8] = "Mage",
+	[9] = "Warlock",
+	[11] = "Druid",
+	-- WotLK:
+	[6] = "Death Knight"
+}
 
 --[[ Global saved variables ]]--
 Hardcore_Settings = {
-	notify = true,
 	level_list = {},
 }
 
+Hardcore_Settings.notify = Hardcore_Settings.notify or true -- Will display death alerts by default.
```

Comment:
This is odd to me, don't we already get the same functionality?

Reply:
Yeah I'll update it to use our existing defaults structure